src/main/java/step3/RacingCarGame.java

```diff
@@ -0,0 +1,51 @@
+package step3;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class RacingCarGame {
+    private List<RacingCar> racingCarList;
+    private Attempt attempt;
+
+    public boolean ready(int carCount, int attemptCount) {
+        racingCarList = new ArrayList<>(carCount);
+        while(racingCarList.size() != carCount) {
```

Comment:
```suggestion
        while (racingCarList.size() != carCount) {
```
코드 포맷팅을 하는 습관을 가져보아요 :)
The auto-formatting shortcut in IntelliJ
For windows Ctrl + Alt + L.
For Mac Command + Option + L.